virtue/virtue-base/kickoff.sh

```diff
@@ -3,5 +3,5 @@
 export XDG_RUNTIME_DIR=~/.xpra
 ~/set-authorized-keys.sh
 mkdir ~/.xpra
-xpra start --bind-tcp=0.0.0.0:2023 --html=on --start-child="$APP_TO_RUN"
+xpra start --bind-ws=0.0.0.0:2023 --start-child="$APP_TO_RUN" --log-file=/home/virtue/xpra.log --exit-with-children
```

Comment:
I am assuming that the `--bind-ws` option somehow enables the html option also so that the xpra client can enable the html client connections?

Reply:
yes, --bind-tcp --htm=on is equivalent to --bind-ws. It's just a shorthand noted here:
https://xpra.org/trac/wiki/Network
For future references, we might want to use --bind-wss, which uses ssl to protect the websocket. I believe that would give a performance increase if we used that instead of the SSH tunnel.

Reply:
Sounds good Thanks